### ISSUE TITLE
Unable to change font size from config.js

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/feedback-widget#7]
From user:

```
There is a small bug in the feedback widget. feedback.css line
634 sets the default font size to 16, but it does it for the
".feedbackTab p" element. The config.js file overrides the parent
".feedbackTab" style, but the "p" element still gets its size from
feedback.css. The end result is it's not possible to change the font
size using config.js.
```

**Comment by c-suh** [comment]
Addressed and awaiting response to [PR](https://github.com/NASA-PDS-Incubator/feedback-widget/pull/8).

**Comment by c-suh** [comment]
Resolved in part 1 of 3 of commit [69ce495b75a9a77edd8794b1a79c0034a89437ba](https://github.com/NASA-PDS-Incubator/feedback-widget/commit/69ce495b75a9a77edd8794b1a79c0034a89437ba).
